scripts/test-cli.sh

```diff
@@ -52,9 +52,21 @@ scalajsld -s -o test.js -mm Foo.main bin
 test -s test.js || fail "scalajsld: empty output"
 test -s test.js.map || fail "scalajsld: empty source map"
 
-node test.js > got.run
+node test.js > got-legacy.run
+cat > want-legacy.run <<EOF
+asdf 2
+EOF
+
+diff got-legacy.run want-legacy.run
+
+mkdir test-output
+scalajsld -s -o test-output -mm Foo.main bin
```

Comment:
It might be nice to use `SplitStyle.SmallestModules` here to make sure the wiring is actually correct.